<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>448</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RandomWords</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{background-color: rgb(167, 204, 237);}

QPushButton{
border-style: solid;
border-color: black;
border-width: 5px;
background-color: rgb(0, 123, 255);
color: rgb(255, 255, 255);
border-radius: 10px;
height: 50px;
}

QPushButton:pressed{
border-style: solid;
border-color: black;
border-width: 5px;
background-color: rgb(66, 189, 255);
color: rgb(255, 255, 255);
border-radius: 10px;
height: 50px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="MainPage">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblMainWord">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(235, 235, 235);</string>
          </property>
          <property name="text">
           <string>Random Word</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnWord1">
          <property name="text">
           <string>Word1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnWord2">
          <property name="text">
           <string>Word2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnWord3">
          <property name="text">
           <string>Word3</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="DisplayPage">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="StatsPage">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>150</y>
          <width>271</width>
          <height>96</height>
         </rect>
        </property>
        <property name="text">
         <string>Stats Page</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="MigratePage">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="lblMigrate">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Migrate from JSON</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="txtJSON">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnLoadJson">
          <property name="text">
           <string>Load words</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tableView_2"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="grpMenu">
      <property name="statusTip">
       <string/>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="accessibleName">
       <string/>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnMain">
         <property name="text">
          <string>Main</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDisplay">
         <property name="text">
          <string>DisplayWords</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnMigrate">
         <property name="text">
          <string>Migrate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnStats">
         <property name="text">
          <string>Stats</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionNew_word"/>
    <addaction name="actionDisplay_words"/>
    <addaction name="actionMigrate_Words"/>
    <addaction name="actionStatystyki"/>
    <addaction name="actionMain"/>
    <addaction name="actionOptions"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_word">
   <property name="text">
    <string>New word</string>
   </property>
  </action>
  <action name="actionDisplay_words">
   <property name="text">
    <string>Display words</string>
   </property>
  </action>
  <action name="actionMigrate_Words">
   <property name="text">
    <string>Migrate Words</string>
   </property>
  </action>
  <action name="actionStatystyki">
   <property name="text">
    <string>Statystyki</string>
   </property>
  </action>
  <action name="actionMain">
   <property name="text">
    <string>Main</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
